guides/common/modules/proc_synchronizing-a-custom-repository.adoc

```diff
@@ -0,0 +1,37 @@
+[id="Synchronizing_a_Custom_Repository_{context}"]
+= Synchronizing a Custom Repository
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} to a disconnected {ProjectServer} via {ISS} (ISS) Network Sync.
```

Comment:
```suggestion
Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} to a disconnected {ProjectServer} through {ISS} (ISS) Network Sync.
```

IMHO we are supposed not to use latin words like "via". Maybe use "using" or "through" instead.